docs/tech-reference/iso-rsync-distributor.rst

```diff
@@ -9,6 +9,13 @@ ssh to perform the file transfer.
 
 Configuration
 =============
+Pulp's SELinux policy includes a ``pulp_manage_rsync`` boolean. When enabled, the
+``pulp_manage_rsync`` boolean allows Pulp to use rsync and make ssh connections. The boolean is
+disabled by default. The ISO Rsync distributor will fail to publish with SELinux Enforcing, unless
```

Comment:
s/Enforcing,/Enforcing/

I suggested the extra comma I think, sorry about that.